Dockerfile.arm

```
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
+WORKDIR /source
+
+# Copy csproj and restore.
+COPY src/Impostor.Server/Impostor.Server.csproj ./src/Impostor.Server/Impostor.Server.csproj
+COPY src/Impostor.Shared/Impostor.Shared.csproj ./src/Impostor.Shared/Impostor.Shared.csproj
+COPY submodules/Hazel-Networking/Hazel/Hazel.csproj ./submodules/Hazel-Networking/Hazel/Hazel.csproj
+
+RUN dotnet restore -r linux-arm ./src/Impostor.Server/Impostor.Server.csproj && \
+    dotnet restore -r linux-arm ./src/Impostor.Shared/Impostor.Shared.csproj && \
+    dotnet restore -r linux-arm ./submodules/Hazel-Networking/Hazel/Hazel.csproj
+
+# Copy everything else.
+COPY submodules/. ./submodules/
+COPY src/. ./src/
+RUN dotnet publish -c release -o /app -r linux-arm --no-restore ./src/Impostor.Server/Impostor.Server.csproj
+
+# Final image.
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:latest
```

```suggestion
FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1-alpine
```

We don't really want to use `latest` here incase a breaking change gets be released in `runtime-deps`.

---

We cannot use the alpine version. I don't know if linux-arm runtime identifier works with alpine image. Maybe using 3.1 is a better solution.
```suggestion
FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1
```